.github/workflows/publish-specs.yml

```diff
@@ -0,0 +1,66 @@
+name: Build, validate and publish spec to GitHub Pages and /TR
+
+# Workflow runs once per spec (WebGPU, WGSL and explainer) on pull requests and
+# on pushes to the main branch that touch the spec. For pull requests, it makes
+# sure that the spec can be generated and that markup is valid. For pushes, it
+# also deploys the generate spec to GitHub Pages and to /TR for WebGPU and WGSL.
+# The "workflow_dispatch" hook allows admins to trigger the workflow manually
+# from GitHub's UI.
+#
+# Workflow uses the w3c/spec-prod action, see base multi-spec repo example at:
+# https://github.com/w3c/spec-prod/blob/main/docs/examples.md#multiple-specs-in-same-repository
+
+on:
+  pull_request:
+    paths:
+      - 'spec/index.bs'
+      - 'wgsl/index.bs'
+      - 'explainer/index.bs'
```

Comment:
Ah, I think no actions are running because of this filter.

We also need to rebuild on changes to `.github/`, as well as any potentially dependent files of each spec (like diagrams and [includes](https://tabatkins.github.io/bikeshed/#bp-include).

I can make this change but please review it.